.drone.yml

```
@@ -189,3 +189,98 @@ trigger:
   event:
     - push
     - pull_request
+---
+kind: pipeline
+type: docker
+name: test-stable-16
+
+steps:
+  - name: test
+    image: ghcr.io/nextcloud/continuous-integration-android8:2
+    privileged: true
+    environment:
+      LOG_USERNAME:
+        from_secret: LOG_USERNAME
+      LOG_PASSWORD:
+        from_secret: LOG_PASSWORD
+      GIT_USERNAME:
+        from_secret: GIT_USERNAME
+      GIT_TOKEN:
+        from_secret: GIT_TOKEN
+    commands:
+      - emulator -avd android -no-snapshot -gpu swiftshader_indirect -no-window -no-audio -skin 500x833 &
+      - sed -i s"#https://server#http://server-oldest-stable#" gradle.properties
+      - sed -i s"#1#5#" ./library/src/androidTest/java/com/owncloud/android/RetryTestRule.kt
+      - ./gradlew assembleDebug
+      - ./scripts/wait_for_emulator.sh
+      - ./scripts/wait_for_server.sh server-oldest-stable
+      - scripts/deleteOutdatedComments.sh "stable" "Unit" $DRONE_PULL_REQUEST $GIT_USERNAME $GIT_TOKEN
+      - scripts/deleteOutdatedComments.sh "stable" "IT" $DRONE_PULL_REQUEST $GIT_USERNAME $GIT_TOKEN
+      - ./gradlew jacocoTestDebugUnitTestReport || scripts/uploadReport.sh $LOG_USERNAME $LOG_PASSWORD $DRONE_BUILD_NUMBER "stable" "Unit" $DRONE_PULL_REQUEST $GIT_USERNAME $GIT_TOKEN
+      - ./gradlew installDebugAndroidTest
+      - ./gradlew createDebugCoverageReport || scripts/uploadReport.sh $LOG_USERNAME $LOG_PASSWORD $DRONE_BUILD_NUMBER "stable" "IT" $DRONE_PULL_REQUEST $GIT_USERNAME $GIT_TOKEN
```

```suggestion
      - ./gradlew jacocoTestDebugUnitTestReport || scripts/uploadReport.sh $LOG_USERNAME $LOG_PASSWORD $DRONE_BUILD_NUMBER "stable16" "Unit" $DRONE_PULL_REQUEST $GIT_USERNAME $GIT_TOKEN
      - ./gradlew installDebugAndroidTest
      - ./gradlew createDebugCoverageReport || scripts/uploadReport.sh $LOG_USERNAME $LOG_PASSWORD $DRONE_BUILD_NUMBER "stable16" "IT" $DRONE_PULL_REQUEST $GIT_USERNAME $GIT_TOKEN
```

This is needed so that the GitHub comments don't conflict for both stable builds